internal/stack/_static/docker-compose-stack.yml.tmpl

```diff
@@ -170,13 +170,14 @@ services:
       interval: 60s
       timeout: 50s
       retries: 5
-    command: bash -c "bin/logstash-plugin install logstash-filter-elastic_integration && logstash -f /usr/share/logstash/pipeline/logstash.conf"
+    command: bash -c 'if [[ ! $(bin/logstash-plugin list) == *"logstash-filter-elastic_integration"* ]]; then echo "Missing plugin logstash-filter-elastic_integration, installing now" && bin/logstash-plugin install logstash-filter-elastic_integration; fi && bin/logstash -f /usr/share/logstash/pipeline/logstash.conf'
```

Comment:
We will need to move this to a script if we continue adding things here :)

Reply:
One we package the plugin (soon with [this PR](https://github.com/elastic/logstash/pull/15769)), we can actually remove this script, no need.